dev-tools/omdb/src/bin/omdb/nexus.rs

```diff
@@ -441,6 +445,45 @@ struct DiskExpungeArgs {
     physical_disk_id: PhysicalDiskUuid,
 }
 
+#[derive(Debug, Args)]
+struct SupportBundleArgs {
+    #[command(subcommand)]
+    command: SupportBundleCommands,
+}
+
+#[derive(Debug, Subcommand)]
+#[allow(clippy::large_enum_variant)]
+enum SupportBundleCommands {
+    /// List all support bundles
+    List,
+    /// Create a new support bundle
+    Create,
+    /// Delete a support bundle
+    Delete(SupportBundleDeleteArgs),
+    /// Download the index of a support bundle
+    ///
+    /// This is a "list of files", from which individual files can be accessed
+    GetIndex(SupportBundleIndexArgs),
+    /// View a file within a support bundle
+    GetFile(SupportBundleFileArgs),
+}
+
+#[derive(Debug, Args)]
+struct SupportBundleDeleteArgs {
+    id: SupportBundleUuid,
+}
+
+#[derive(Debug, Args)]
+struct SupportBundleIndexArgs {
+    id: SupportBundleUuid,
+}
+
+#[derive(Debug, Args)]
+struct SupportBundleFileArgs {
+    id: SupportBundleUuid,
+    path: Utf8PathBuf,
```

Comment:
We'll eventually have non-text files in bundles, e.g. core dumps, SP dumps. An option to directly save the file would be useful for these.

```suggestion
    path: Utf8PathBuf,
    output_path: Utf8PathBuf,
```

Reply:
Added, also, removed utf8 parsing so the core dump cases can be passed through as regular binary files.